src/main/resources/assets/opencomputers/loot/openos/bin/flash.lua

```diff
@@ -53,8 +57,6 @@ local function writeRom()
     io.write("Beginning to flash EEPROM.\n")
   end
 
-  local eeprom = component.eeprom
-
```

Comment:
This breaks the ability of an EEPROM to be inserted after starting the program, but before flashing.